custom/opm/reports.py

```diff
@@ -888,7 +896,7 @@ def row_objects(self):
         self._debug_data = []
         for index, row in enumerate(self.get_rows(self.datespan), 1):
             try:
-                rows.append(self.get_row_data(row, index=index))
+                rows.append(self.get_row_data(row, index=1))
```

Comment:
(offlined with @sravfeyn this makes sense and this line can actually be deleted)